### PR TITLE
Design6

### PR DESCRIPTION
**Pull request by grijos**

### CODE REVIEW

**Comment by nicosampler** [comment]
@grijos 

image

- [ ] Can we add a little padding here? In the cellphone, it's too close to the left and right sides. 

image

- [ ] Something is broken there.

- [ ] Also, we should not show the shop id as a number, we should use the encoded one (letters) I can change it.

File: pages_/my-shop.tsx [hunk 1]
```diff
@@ -257,19 +268,23 @@ const MyShop = ({ isLoggedIn, shopId }: Props) => {
...
               className="d-flex justify-content-between align-items-center py-2"
             >
               <FontAwesomeIcon icon={faPrint} fixedWidth className="mr-2" />
               {t('common:print-poster')}
               <div></div>
             </Button>
           </Link>
+
+          <p className="mt-2 mb-0">
+            <small>https://cauda.app/{encodeId(shopId)}</small>
```
Comment (ramirotw) [inline pages_/my-shop.tsx:286]: @nicosampler the shopId encoding should be done in the getServerSideProps function before sending it as props to the component so the hashids lib wouldn't be added to the browser bundle

File: pages_/turn/[turnId].tsx [hunk 2]
```diff
@@ -130,6 +130,7 @@ const MyTurn = ({
             {t('common:my-turn')}
           </Button>
 
+          {/* TODO NICO: Add "myturn__number--disabled" class for disabled myturn__number */}
```
Comment (ramirotw) [inline pages_/turn/[turnId].tsx:133]: is this comment still valid?

Comment (grijos) [reply pages_/turn/[turnId].tsx:133]: @ramirotw Yes. For "expired" or already completed Turns we need to add that class to the number to change its style. It is not ok to keep it as a blue number. We need to also add some wording stating that "Este turno está completado" or something like that.

Comment (grijos) [reply pages_/turn/[turnId].tsx:133]: @ramirotw Right now, the only indication of Turn status is in the Home turn list. We also need some visual indication of turn-expiration in the My Turn screen as well.

**Pull request merged**